onnxruntime/core/framework/session_state.h

```diff
@@ -421,6 +423,12 @@ class SessionState {
   // subset of initialized_tensors_ that are constant and cannot be overridden at runtime
   std::unordered_map<int, OrtValue> constant_initialized_tensors_;
 
+  // This is an auxillary lookup to check if the OrtValue was actually a sparse tensor
```

Comment:
>auxillary [](http://example.com/codeflow?start=16&length=9)

"auxillary" -> "auxiliary"
nit: I guess "bc" means "because" - if so, can we just say "because"?